macros/CSS_key_concepts.ejs

```diff
@@ -75,6 +75,16 @@ switch(env.locale) {
  <a href="/ru/docs/Web/CSS/Shorthand_properties" title="Сокращённые свойства">сокращённые свойства</a> и
  <a href="/ru/docs/Web/CSS/Замещаемый_элемент" title="Замещаемые элементы">замещаемые элементы</a>.<%
         break;
+    case "zh-CN":
+%>CSS Key Concepts:
+ <a href="/zh-CN/docs/Web/CSS/Syntax" title="语法">CSS 语法</a>,
+ <a href="/zh-CN/docs/Web/CSS/At-rule" title="@规则">@规则</a>,
+ <a href="/zh-CN/docs/Web/CSS/Comments" title="注释">注释</a>,
+ <a href="/zh-CN/docs/Web/CSS/Specificity" title="优先级">优先级</a>和<a href="/zh-CN/docs/Web/CSS/inheritance" title="继承">继承</a>,
+ the <a href="/zh-CN/docs/Web/CSS/CSS_Box_Model/Introduction_to_the_CSS_box_model" title="盒模型">盒模型</a>,
+ <a href="/zh-CN/docs/Web/CSS/Layout_mode" title="CSS 布局模式">布局模式</a>和<a href="/zh-CN/docs/Web/Guide/CSS/Visual_formatting_model" title="视觉格式化模型">视觉格式化模型</a>，以及<a href="/zh-CN/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing" title="外边距合并">外边距合并</a>，或者<a href="/zh-CN/docs/Web/CSS/initial_value" title="初始值">初始</a>、<a href="/zh-CN/docs/Web/CSS/computed_value" title="计算值">计算</a>、<a href="/zh-CN/docs/Web/CSS/resolved_value" title="解析值">解析</a>、<a href="/zh-CN/docs/Web/CSS/specified_value" title="指定值">指定</a>、<a href="/zh-CN/docs/Web/CSS/used_value" title="使用值">使用</a>和<a href="/zh-CN/docs/Web/CSS/actual_value" title="实际值">实际</a>值。
+ Definitions of <a href="/zh-CN/docs/Web/CSS/Value_definition_syntax" title="值定义语法">值语法</a>、<a href="/zh-CN/docs/Web/CSS/Shorthand_properties" title="简写属性">简写属性</a>和<a href="/zh-CN/docs/Web/CSS/Replaced_element" title="可替换元素">可替换元素</a>。<%
+        break;
```

Comment:
👍 tested on [Web/CSS/initial_value](https://developer.mozilla.org/zh-CN/docs/Web/CSS/initial_value). It also works in French, despite GitHub not liking the format...